micromagneticdata/drive.py

```diff
@@ -415,3 +416,25 @@ def slider(self, description="step", **kwargs):
         return ipywidgets.IntSlider(
             value=0, min=0, max=self.n - 1, step=1, description=description, **kwargs
         )
+
+    def to_xarray(self):
+        """Export ``micromagneticdata.Drive`` as ``xarray.Dataset``"""
+        if self.info["driver"] == "MinDriver":
+            darray = self[0].to_xarray()
+        else:
+            field_darrays = (self[i].to_xarray() for i in range(self.n))
+            if self.info["driver"] == "TimeDriver":
+                darray = xr.concat(field_darrays, dim="t").assign_coords(
+                    {"t": self.table.data["t"].to_numpy()}
+                )
+            else:
+                darray = xr.concat(field_darrays, dim="B_hysteresis").assign_coords(
+                    {
+                        f"B{i}": ("B_hysteresis", self.table.data[f"B{i}_hysteresis"])
+                        for i in "xyz"
+                    }
+                )
```

Comment:
The naming is inconsistent
- Sometimes components are called `Bx_hysteresis` and sometimes `Bx` etc.
- And there currently is the difference `B_hysteresis` and `Bx` for a component.

Reply:
I'll change it to `Bx_hysteresis` to be consistent.